taxi/forms.py

```diff
@@ -0,0 +1,80 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from django.core.validators import MinLengthValidator, MaxLengthValidator
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number",)
+
+    license_number = forms.CharField(
+        required=True,
+        validators=[MinLengthValidator(8), MaxLengthValidator(8)]
+    )
```

Comment:
You declared the `license_number` field and validation 2 times: in the `DriverCreationForm` and `DriverLicenseUpdateForm` classes. This doesn't comply with the **DRY** principle.
Consider declaring this field in a separate class and inheriting it as a **Mixin** class.